system.go

```diff
@@ -251,7 +251,9 @@ func getiOSVersion(uaPlatformGroup string) int {
 	if len(s) == 0 {
```

Comment:
This check should change since we're slicing below from len(s)-2, so if that ends up being negative it'll panic.